### ISSUE TITLE
Off Canvas Cart

### ISSUE THREAD

**Issue by lojapratica** [CrocoBlock/suggestions#2938]
I would like to suggest 2 native functions for JetWooBuilder:

1. https://codecanyon.net/item/woocommerce-cart-all-in-one/30184317
It is an Off Canvas that can be configured to appear whenever a product is added to the cart. Or as a second option the addition of an element that we can include within the popup where we display the products in the cart in the same way that appears in the mentioned plugin. And I also know that there is an element that works like this, but I wanted the off canvas to appear whenever a product is added to the cart.

2. https://codecanyon.net/item/woocommerce-free-shipping-bar/19536343
This plugin explains itself

Not only me, but many depend a lot on this function because we see many big stores with this function

**Comment by 79ho3ein** [comment]
Yes, the shopping cart on the menu is really needed for jet woobuilder.
If it is added, it will be really great.
Please check

thanks

**Comment by UraraReika** [comment]
Hello @lojapratica ! 
1 - You can achieve this with the help of JetPopup plugin. You can style the popup whatever you want and then https://prnt.sc/117bdbw use in with this option. This option control exist in every widget connected with add to cart functionality.

2 - We will consider the need for this functionality and possibly add it in future updates.

**Comment by lojapratica** [comment]
Yes, I even thought of inserting an element with the products added to the cart there inside the ppup, but there is no one that is 'small' according to the print
https://prnt.sc/117csf7

**Comment by UraraReika** [comment]
View of minicart item depends of theme styling. For now we don't give an opportunity to style such an item.

**Comment by lojapratica** [comment]
Thank you my friend, I want to leave this suggestion

**Comment by Hans-101** [comment]
+1

**Comment by P-happiness** [comment]
Guys, just to give you some inspirations...

Look at this: https://cartpops.com/

I hope you'll be able to do something like this!
I know it's a huge job, but this is a game changer!
